packages/react/src/toast/useToast.ts

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { ToastContext } from './provider/ToastProviderContext';
+
+/**
+ * Returns the array of toasts and methods to create toasts.
+ */
+export function useToast(): useToast.ReturnValue {
```

Comment:
It feels a bit off to call `useToast` and receive functions that operate on the whole collection of toasts. IMO `useToasts` would work better here (or `useToaster`, but that's more contentious)

Reply:
How about `useToastManager`? This matches the Global API. Maybe `Toast.Manager` would be better than `Provider`

Reply:
Yeah, it would work too.